sdk/storage/azure-storage-queue/azure/storage/queue/_queue_service_client.py

```diff
@@ -247,6 +238,8 @@ def set_service_properties( # type: ignore
         :type cors: list(~azure.storage.queue.CorsRule)
         :keyword int timeout:
             The timeout parameter is expressed in seconds.
+        :returns: None
+        :rtype: None
```

Comment:
I didn't think so either, but iirc it was complaining? I was also very confused because within the same file you'll see another function that returns `None`, doesnt' have an `rtype` and it didn't complain. Maybe I did a bad job of making a change and then not saving before running PyLint, so will give it another shot